tools/stations2geojson.rb

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+
+require 'csv'
+require 'json'
+
+# A script to create a mappable edition of the stations.csv database.
+# To create GeoJSON from `stations.csv` download the stations.csv file and pipe through
+# this script:
+#	curl https://raw.githubusercontent.com/capitainetrain/stations/master/stations.csv > stations.csv
+#	to_geojson.rb < stations.csv > stations.geojson
+# This script is designed to be short and simple. No error or conformance checking is
+# performed.
+
+features = []
+
+CSV.new(STDIN, :headers => true, :col_sep => ";", :encoding => "UTF-8").each do |station|
+  if station["longitude"] && station["latitude"]
+    features << {
+      :type => "Feature",
+      :geometry => {
+        :type => "Point",
+        :coordinates => [station["longitude"].to_f, station["latitude"].to_f]
+      },
+      :properties => {
+        :name => station["name"]
```

Comment:
As a side-note, the original Perl code included all the station data, not just the name. This is cleaner.